lib/ups.rb

```diff
@@ -0,0 +1,18 @@
+class Ups
+  attr_accessor :ups_audit
+  
+  def initialize
+    @ups_audit = ActiveShipping::UPS.new(login: ENV["UPS_LOGIN"], password: ENV["UPS_PASSWORD"], key: ENV["UPS_KEY"])
+  end
+
+  def find_rates(shipment_params)
+    origin = ActiveShipping::Location.new(country: 'US', state: shipment_params["origin"]["state"], city: shipment_params["origin"]["city"], zip: shipment_params["origin"]["zip"])
```

Comment:
To make these lines a bit more readable, you could could do something like this: 

`origin = ActiveShipping::Location.new(country: 'US', state: shipment_params["origin"]["state"], 
                                                                     city: shipment_params["origin"]["city"], 
                                                                      zip: shipment_params["origin"]["zip"])`